lib/fetch.go

```diff
@@ -203,6 +222,11 @@ func (of *OCIFetcher) fetchManifest(u *URL) (*schema.ImageManifest, error) {
 		return nil, fmt.Errorf("unexpected http code: %d, URL: %s", res.StatusCode, req.URL)
 	}
 
+	digest := res.Header.Get("Docker-Content-Digest")
```

Comment:
I don't believe that this header is available on every docker registry (quay is one lacking it). It would be best to generate this here if possible, but I'm not actually sure how this digest is generated.